stable/spark/templates/spark-livy-hpa.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.Livy.Autoscaling.Enabled }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  labels:
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: "{{ .Release.Name }}-{{ .Values.Livy.Component }}"
+  name: {{ template "livy-fullname" . }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1beta1
+    kind: Deployment
+    name: {{ template "livy-fullname" . }}
+  minReplicas: {{ .Values.Livy.Replicas }}
+  maxReplicas: {{ .Values.Livy.ReplicasMax }}
```

Comment:
This is another one of those things that pains me to point out because I want this to be a little... ehrm.... more automated, but I'm pretty sure you missed documenting ReplicasMax in the README.

Reply:
will fix. thanks